config/jupyterhub_config.py

```diff
@@ -19,7 +19,10 @@
 # Set the authenticator class to nativeauthenticator
 # ref: https://native-authenticator.readthedocs.io/en/latest/quickstart.html
 c.JupyterHub.authenticator_class = 'native'
-c.JupyterHub.template_paths = [f"{os.path.dirname(nativeauthenticator.__file__)}/templates/"]
+c.JupyterHub.template_paths = [
+    os.path.abspath("/templates"),
```

Comment:
```suggestion
    os.environ['JUPYTERHUB_TEMPLATES_DIR'],
```

Reply:
in Dockerfile, add below before `# Jupyter Hub user home directory`

```# Set up JupyterHub login templates directory
ENV JUPYTERHUB_TEMPLATES_DIR=/templates
RUN mkdir -p ${JUPYTERHUB_TEMPLATES_DIR}
COPY ./templates/ ${JUPYTERHUB_TEMPLATES_DIR}
```

Reply:
Sweet!!